align/pnr/render_placement.py

```diff
@@ -135,7 +135,7 @@ def scalar_rational_scaling( v, *, mul=1, div=1):
         q, r = divmod( mul*v, div)
         #assert r == 0
         if r != 0:
-            logger.error(f'Value {v} not a whole number after scaling: q={q} r={r} mul={mul} div={div}')
+            logger.debug(f'Value {v} not a whole number after scaling: q={q} r={r} mul={mul} div={div}')
```

Comment:
@stevenmburns , is this okay to do so? This error is only triggered for leaves in `placer.py` and leaves do not seem to be used in the rest of the code:
```
        hack_placement_verilog_d = scale_placement_verilog( placement_verilog_d, 2*scale_factor, invert=True)        
```

Reply:
The motivation is to eliminate large number of false errors due when wires can start and stop at half-nanometer increments.